src/main/io/gps.h

```diff
@@ -35,6 +35,32 @@
 #define GPS_Y 0
 #define GPS_MIN_SAT_COUNT 4      // number of sats to trigger low sat count sanity check
 
+#ifdef USE_GPS_UBLOX
+typedef enum {
+    UBX_VERSION_UNDEF = 0,
```

Comment:
Maybe it would be better to add UBX_VERSION_UNKNOWN, to handle state where version message was received, but is not in table

Reply:
The struct suggests that undefined includes unknown.  Is that sufficient?
```
struct ubloxVersion_s ubloxVersion_map[] = {
        [UBX_VERSION_UNDEF] = {~0, "UNKNOWN"},
```

Reply:
@ledvinap may I mark this as resolved, since I do not completely know what to do, and it works OK as is